tools/src/com.oracle.truffle.tools.agentscript/src/com/oracle/truffle/tools/agentscript/impl/InsightFilter.java

```diff
@@ -49,6 +49,8 @@ final class InsightFilter {
     private static final String CONFIG_EXPRESSIONS = "expressions";
     private static final String CONFIG_STATEMENTS = "statements";
     private static final String CONFIG_ROOTS = "roots";
+    private static final String CONFIG_READ_VARIABLES = "reads";
+    private static final String CONFIG_WRITE_VARIABLES = "writes";
```

Comment:
- I was thinking of `variableReads`, etc.
- but at the end I opted for simple `reads` and `writes`
- they rhyme well with `statements`, `roots`, etc.